From 2e0446080c561b99a31f37e9370913cf95f0bb42 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Fri, 12 Sep 2014 18:10:44 -0400
Subject: [powerpc] Ignore .TOC. relocations

Message-id: <1410545655-205645-416-git-send-email-dzickus@redhat.com>
Patchwork-id: 94631
O-Subject: [RHEL7 PATCH 415/626] powerpc: Ignore .TOC. relocations
Bugzilla: 1127366
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: David Gibson <dgibson@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1127366

commit 814e4cd98f777c7fa3b42e0468030cd341fb8b6b
Author: Anton Blanchard <anton@samba.org>
Date:   Tue Feb 4 16:08:02 2014 +1100

    powerpc: Ignore .TOC. relocations

    The linker fixes up .TOC. relocations, so prom_init_check.sh should
    ignore them.

    Signed-off-by: Anton Blanchard <anton@samba.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/powerpc/kernel/prom_init_check.sh b/arch/powerpc/kernel/prom_init_check.sh
index 3765da6..fe4bcd6 100644
--- a/arch/powerpc/kernel/prom_init_check.sh
+++ b/arch/powerpc/kernel/prom_init_check.sh
@@ -22,7 +22,8 @@ __secondary_hold_acknowledge __secondary_hold_spinloop __start
 strcmp strcpy strlcpy strlen strncmp strstr logo_linux_clut224
 reloc_got2 kernstart_addr memstart_addr linux_banner _stext
 opal_query_takeover opal_do_takeover opal_enter_rtas opal_secondary_entry
-boot_command_line __prom_init_toc_start __prom_init_toc_end"
+boot_command_line __prom_init_toc_start __prom_init_toc_end
+TOC."
 
 NM="$1"
 OBJ="$2"
-- 
1.7.1